examples/subplots_axes_and_figures/axes_margins.py

```diff
@@ -32,3 +32,57 @@ def f(t):
 ax3.set_title('Zoomed in')
 
 plt.show()
+
+
+#############################################################################
+#
+# On the "stickiness" of certain plotting methods
```

Comment:
I think it might be worth mentioning that this example now discusses the "stickiness" features in the top description of the plot and maybe even the title.

Reply:
Excellent suggestion. Added a blurb. Let me know what you think.

Reply:
It looks good! If travis is happy, I'll go ahead and merge!